src/System.Security.Principal.Windows/tests/WindowsIdentityTests.cs

```diff
@@ -124,6 +124,20 @@ public static void CheckUserClaims()
         }
     }
 
+    [Fact]
+    public static void RunImpersonatedTest_InvalidHandle()
+    {
+        Assert.Throws<ArgumentException>(() =>
```

Comment:
Nit: consider moving this to just wrap the RunImpersonated call rather than the whole using